.idea/php.xml

```diff
@@ -219,13 +219,15 @@
       </server>
     </servers>
   </component>
-  <component name="PhpProjectSharedConfiguration" php_language_level="7.3" />
+  <component name="PhpProjectSharedConfiguration" php_language_level="8.0">
+    <option name="suggestChangeDefaultLanguageLevel" value="false" />
+  </component>
   <component name="PhpUnit">
     <phpunit_settings>
       <phpunit_by_interpreter interpreter_id="4ed8db31-a6d2-4689-868b-5471721ce351" custom_loader_path="/app/vendor/autoload.php" phpunit_phar_path="" />
       <phpunit_by_interpreter interpreter_id="881ee65e-dab4-4d27-9858-db7669d088aa" custom_loader_path="/app/vendor/autoload.php" phpunit_phar_path="" />
       <phpunit_by_interpreter interpreter_id="64d769c9-e591-4551-b455-0f16f76b2498" custom_loader_path="/app/vendor/autoload.php" phpunit_phar_path="" />
-      <PhpUnitSettings custom_loader_path="$USER_HOME$/3_opg-lpa/service-pdf/vendor/autoload.php" phpunit_phar_path="" />
+      <PhpUnitSettings custom_loader_path="$USER_HOME$/3_opg-lpa/service-api/vendor/autoload.php" phpunit_phar_path="" />
```

Comment:
Any idea why PHPStorm did this?  ;)

Reply:
I'd happily take out the PHPStorm config. I just use it for debugging occasionally so I don't really care how it's configured. I don't think it should be in the repo at all to be honest.

Reply:
If it is not in the repo, how would we have shared config between the team though?  (Fair enough at the moment we may be trampling on each other )